sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -380,6 +380,11 @@ class SQLContext(@transient val sparkContext: SparkContext)
     jsonRDD(json.rdd, schema)
   }
 
+  @Experimental
+  def inferJsonSchema(json: String): StructType = {
```

Comment:
How about we close it for now? Also, feel free to leave comment to that JIRA. Thank you.